examples/frigate/frigate.ini

```diff
@@ -0,0 +1,56 @@
+[defaults]
+functions = frigate.py
+launch    = apprise-ntfy, store-jpeg
+
+status_publish = True
+
+
+
+# =====================
+# Frigate event to Ntfy
+# =====================
+
+# Format: JSON
+# Docs:   https://docs.frigate.video/integrations/mqtt/#frigateevents
+
+
+
+[config:apprise-ntfy]
+module   = apprise_single
+baseuri  = ntfys://ntfy/frigate
+filename_template = './var/media/{camera}-{label}.jpg'
```

Comment:
See also my comment above, at https://github.com/jpmens/mqttwarn/pull/635#discussion_r1166670893. This would use the [another code path](https://github.com/binwiederhier/ntfy/blob/1ccf6597815b7b3bd7a9b7f30a6d6d60c4620798/server/server.go#L826-L828) instead.

Reply:
Please consider this comment as obsolete, see https://github.com/jpmens/mqttwarn/pull/635#discussion_r1166789969.